Tests/ClientRuntimeTests/NetworkingTests/URLSession/FoundationStreamBridgeTests.swift

```diff
@@ -65,7 +65,12 @@ class FoundationStreamBridgeTests: XCTestCase {
 
             // Create a stream bridge with our original data & open it
             let bufferedStream = BufferedStream(data: originalData, isClosed: true)
-            let subject = FoundationStreamBridge(readableStream: bufferedStream, bridgeBufferSize: bridgeBufferSize, boundStreamBufferSize: boundStreamBufferSize, logger: TestLogger())
+            let subject = FoundationStreamBridge(
+                readableStream: bufferedStream,
+                bridgeBufferSize: bridgeBufferSize,
+                boundStreamBufferSize: boundStreamBufferSize,
+                logger: TestLogger(),
+                telemetry: HttpTelemetry(httpScope: "FoundationStreamBridgeTests"))
```

Comment:
Definitely can be changed: wiring in telemetry and logger seems redundant, and maybe there should be a default `HttpTelemetry` implementation for `FoundationStreamBridge`.